services/job-scheduler/model/worker.go

```diff
@@ -0,0 +1,52 @@
+package model
+
+import (
+	"fmt"
+	"net/url"
+)
+
+type WorkerStatus string
+
+const (
+	WorkerStatusAvailable WorkerStatus = "AVAILABLE" // default value for new worker
+	WorkerStatusRunning   WorkerStatus = "RUNNING"   // set by Job Scheduler
+)
+
+type Worker struct {
+	Id     string       `json:"id"` // FIXME: actually UUID
+	Status WorkerStatus `json:"status"`
+	Zone   string       `json:"zone"`
+}
+
+// -------------------------- Endpoints --------------------------
+
+func PutWorkerStatusEndpoint(base string, id string) string {
+	// FIXME: change string to UUID
+	return fmt.Sprintf("%s/workers/%s/status", base, id)
+}
+
+func GetWorkersEndpoint(base string) string {
+	baseUrl := fmt.Sprintf("%s/workers", base)
+
+	params := url.Values{}
+	params.Add("status", string(WorkerStatusAvailable))
+
+	fullUrl := baseUrl + "?" + params.Encode()
+	return fullUrl
+}
+
+// -------------------------- Response & Request --------------------------
+
+type GetWorkersResponse []Worker
+
+// This struct is used for the patch-request to the worker service
+type UpdateWorkerPayload struct {
+	WorkerStatus WorkerStatus `json:"workerStatus"` // default (and probably only) value is "running"
+}
```

Comment:
Worker Registry is using `json:"status"` as accepted Requestpayload value right now.
```suggestion
type UpdateWorkerPayload struct {
	WorkerStatus WorkerStatus `json:"status"` // default (and probably only) value is "running"
}
```
Screenshot from worker registry model:
![image](https://github.com/user-attachments/assets/88b4bd0e-18df-49f5-b52f-ad6218a4c7dc)